etc/module.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0"?>
 <config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:Module/etc/module.xsd">
-    <module name="Acquired_Payments" setup_version="0.9.5">
+    <module name="Acquired_Payments" setup_version="0.9.6">
```

Comment:
Is this something we actually need to increment every release? Or should it only be incremented when something in here changes?

- [ ] What's the versioning process for `setup_version`?

Reply:
i just kept it 1:1 to the version so far, but no, we only need to raise it on database changes. some modules keep it in line with composer version others keep it separate, it is a design choice rather than a hard requirement of keeping in sync.